apps/lfx-one/src/app/modules/meetings/meetings-dashboard/meetings-dashboard.component.html

```
@@ -20,6 +20,143 @@ <h1 class="font-display font-light text-2xl">{{ activeLens() === 'me' ? 'My Meet
+            <div class="flex items-center gap-3 p-4">
+              <div class="flex items-center justify-center w-11 h-11 rounded-full bg-gray-200 text-gray-500 flex-shrink-0">
+                <i class="fa-light fa-clock-rotate-left text-xl"></i>
+              </div>
+              <div class="flex flex-col gap-0.5">
+                @if (meLensStatsLoading()) {
+                  <p class="text-xl font-medium text-gray-400">&mdash;</p>
+                } @else {
+                  <p class="text-xl font-medium text-gray-900">{{ pastThisMonthCount() }}</p>
+                }
+                <p class="text-sm font-normal text-gray-900">Past This Month</p>
+                @if (!meLensStatsLoading() && pastThisMonthCount() > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)